pdns/pdns.go

```diff
@@ -0,0 +1,35 @@
+// Package pdns adapts the lego PowerDNS
+// provider for Caddy. Importing this package plugs it in.
+package pdns
+
+import (
+	"errors"
+	"net/url"
+
+	"github.com/mholt/caddy/caddytls"
+	"github.com/xenolf/lego/providers/dns/pdns"
```

Comment:
While we're on the subject of spelling it out, do you think the package should be "powerdns" too? (I mean, in case there's some other DNS provider that starts with a P)

Reply:
I used the name as given in `lego`. Further, it's common for PowerDNS to be referred to as `pdns`, as for example, the binary for the server is `pdns_server`.